lib/ui/octicon.jsx

```diff
@@ -64,7 +64,7 @@ export class Octicon {
 			classNames.push('disabled');
 		}
 
-		const fill = disabled ? '#505868' : '#9da5b4';
+		const fill = getComputedStyle(document.querySelector("atom-panel")).color;
```

Comment:
<!--
  3 failure:  lib/ui/octicon.js..., lib/ui/octicon.js..., lib/ui/octicon.js...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: lib/ui/octicon.jsx;
  Line: 67;
-->
- :no_entry_sign: lib/ui/octicon.jsx line 67 – 'document' is not defined. (no-undef)
- :no_entry_sign: lib/ui/octicon.jsx line 67 – 'getComputedStyle' is not defined. (no-undef)
- :no_entry_sign: lib/ui/octicon.jsx line 67 – Strings must use singlequote. (quotes)